camdaux/views/vw_allowance_based_compliance_bulk_files_to_generate.sql

```diff
@@ -9,4 +9,11 @@ CREATE OR REPLACE VIEW camdaux.vw_allowance_based_compliance_bulk_files_to_gener
    FROM camddmw.account_compliance_dim acd
      JOIN camddmw.account_fact af ON acd.account_number::text = af.account_number::text AND acd.prg_code::text = af.prg_code::text
      JOIN camdmd.program_code pc ON acd.prg_code::text = pc.prg_cd::text AND pc.bulk_file_active = 1::numeric
-  WHERE (acd.last_update_date >= date(timezone('est'::text, CURRENT_TIMESTAMP) - '1 day'::interval) OR af.last_update_date >= date(timezone('est'::text, CURRENT_TIMESTAMP) - '1 day'::interval)) AND camdaux.can_generate_compliance(acd.op_year::integer, acd.prg_code);
+      JOIN (
+              SELECT  coalesce(max(job.Start_Date), current_timestamp) AS Last_Start_Timestamp
```

Comment:
Change "current_timestamp" to "timezone('est'::text, current_timestamp) - '90 day'::interval".
- The default timestamp should be for 90 days ago.
- The timezone adjustment does not do anything because we are already EST, but is probably a better practice.